src/module/apps/sidebar/actor-directory.ts

```diff
@@ -87,16 +87,17 @@ class ActorDirectoryPF2e extends fa.sidebar.tabs.ActorDirectory<ActorPF2e<null>>
     }
 
     override async _onRender(context: object, options: HandlebarsRenderOptions): Promise<void> {
-        await super._onRender(context, options);
-        const html = this.element;
-
         // Move the party list into the directory part
+        // This must occur before super._onRender() so that drag/drop is registered correctly
         if (options.parts.includes("directory")) {
             const partiesPart = this.parts["parties"];
             partiesPart.remove();
             this.parts["directory"].prepend(partiesPart);
         }
 
+        await super._onRender(context, options);
+        const html = this.element;
+
         // Strip actor level from actors we lack proper observer permission for
         for (const element of htmlQueryAll(html, "li.directory-item.actor")) {
```

Comment:
```suggestion
        for (const element of htmlQueryAll(this.element, "li.directory-item.actor")) {
```
Since it's a one-time use